src/hera.cpp

```diff
@@ -44,6 +44,47 @@ using namespace HeraVM;
 
 namespace {
 
+vector<uint8_t> sentinel(struct evm_context* context, vector<uint8_t> const& input)
+{
+#if HERA_DEBUGGING
+  cout << "Metering (input " << input.size() << " bytes)..." << endl;
+#endif
+
+#if HERA_METERING_CONTRACT
+  evm_message metering_message = {
+    .address = { .bytes = { 10 } }, // precompile address 0x00...0a
```

Comment:
The address is an array of bytes (aka "big endian"), so it must be `{0, 0, 0, ... , 10}`.